test/placement/placement_test.cpp

```diff
@@ -30,9 +47,37 @@ TEST_CASE_METHOD(NetlistFixture, "Placement: placing a cell", "[placement]") {
     ophidian::util::Location cell2Location(20, 10);
     placement.placeCell(cell2, cell2Location);
 
-    ophidian::util::Location placedCell1Location = placement.location(cell1);
-    ophidian::util::Location placedCell2Location = placement.location(cell2);
+    ophidian::util::Location placedCell1Location = placement.cellLocation(cell1);
+    ophidian::util::Location placedCell2Location = placement.cellLocation(cell2);
     REQUIRE(cell1Location == placedCell1Location);
     REQUIRE(cell2Location == placedCell2Location);
     REQUIRE(!(placedCell1Location == placedCell2Location));
 }
+
+TEST_CASE_METHOD(NetlistFixture, "Placement: placing an input pad", "[placement]") {
```

Comment:
I see. It does make sense. Although we need a smart mechanism for getting the location of a given pin in two cases:

1. the pin is related to a pad: return this property you just created.
2. the pin has an owner cell: return its offet + owner's location.